mods/ra/maps/allies-05a/AI.lua

```diff
@@ -35,19 +40,45 @@ SetupAttackGroup = function()
 end
 
 SendAttack = function()
-	if not Attacking then
+	if Attacking then
 		return
 	end
-	Attacking = false
-	HoldProduction = false
+	Attacking = true
+	HoldProduction = true
 
-	local units = SetupAttackGroup()
-	Utils.Do(units, function(unit)
-		IdleHunt(unit)
-	end)
+	local units = { }
+	if SendWaterTransports and Utils.RandomInteger(0,1) == 1 then
```

Comment:
This will never happen, because `Utils.RandomInteger` never returns the high value. Yes, it's annoying, but it's too late to change now.